src/Migrations/Version20221213144526.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MatchBot\Migrations;
+
+use Doctrine\DBAL\Schema\Schema;
+use Doctrine\Migrations\AbstractMigration;
+
+final class Version20221213144526 extends AbstractMigration
+{
+    public function getDescription(): string
+    {
+        return 'Remove gift aid added to donations by mistake';
+    }
+
+    public function up(Schema $schema): void
+    {
+        $this->addSql('UPDATE Donation set giftAid = 0 where Donation.salesforceId IN ("a066900001vWmAaAAK", "a066900001vWlRHAA0", "a066900001vVkwZAAS") LIMIT 3');
```

Comment:
@bdsl I think you might be missing some more fields to update. See the similar PR made for MAT-245:
https://github.com/thebiggive/matchbot/pull/382/files

Reply:
@alihejazi Thanks. https://github.com/thebiggive/matchbot/pull/462/commits/84f43057f5cf913d148e0427d73488108feeb55a should fix it.